.gitignore

```diff
@@ -12,7 +12,9 @@ db.sqlite3-journal
 /media
 /static
 .env
-
+.idea
+env/
```

Comment:
added option to ignore virtual environment files and pycharm temporary generated directories